packages/core/src/components/results/ResultsExamQuestionManualScore.tsx

```diff
@@ -40,23 +40,47 @@ function ResultsExamQuestionManualScore({ scores, maxScore, displayNumber }: Res
     ...(scores.censoring ? normalizeCensoringScores(scores.censoring) : []),
     scores.pregrading && normalizePregradingScore(scores.pregrading)
   ].filter(s => s) as NormalizedScore[]
+
   const containerProps = { answers, displayNumber }
   return (
     <ResultsExamQuestionScoresContainer {...containerProps}>
-      {maxScore || null} <Translation>{t => t('points-max')}</Translation>
       {normalizedScores.map((score, i) => (
-        <div key={i}>
-          <span>{i === 0 ? <b>{score.score} p</b> : score.score + ' p'} </span>
-          <span>{score.shortCode} </span>
-          <span>
-            (<Translation>{t => t(score.type)}</Translation>)
-          </span>
-        </div>
+        <ScoreRow key={i} {...score} latest={i === 0} maxScore={maxScore} />
       ))}
     </ResultsExamQuestionScoresContainer>
   )
 }
 
+interface ScoreRowProps {
+  maxScore?: number
+  latest: boolean
+}
+
+function ScoreRow({ score, shortCode, type, maxScore, latest }: ScoreRowProps & NormalizedScore) {
+  return (
+    <div className={`${!latest && 'e-color-grey'}`}>
+      <ScoreColumn className={`${latest && 'e-font-size-m'}`}>
+        {latest ? <b>{score}</b> : score}
```

Comment:
Lisäyksenä tähän AutoScore vs ManualScore keskusteluun, AutoScoressa on placeholderi `_` puuttuvalle arvostelulle. Halutaankohan se ManualScoreen myös? Jos joo, se olisi vahvempi syy yhdistellä niitä kahta (Riittäisikö jopa yksi Score??)

Reply:
Jutellaan porukalla näistä esim. huomenna meetissä. Näitä voidaan mahdollisesti generoida uudet taskit jos tarvitsee. Mergetään nyt as is.